js/core/tv.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * TV Simulator '99
+ * @Author: zshall 
+ * @Date: 2017-05-02 22:34:58 
+ * @Last Modified by: zshall
+ * @Last Modified time: 2017-05-03 00:31:21
+ */
+const maxWidth = 1200;
+const maxHeight = 1000;
+const warmupTime = 3000;
+var firstStart = true;
+const classes = {
+    Channel12
+};
+
+$(document).ready(() => {
+    if( !Helpers.isMobile().any() ){
+        $(window).resize(function() {
+            handleResize();
+        });
+        $('.screen').addClass('scanlines');
+    }
+
+    handleResize();
+    
+    // about
+    $('#about-button').click(() => {
```

Comment:
Might want to stick to a single inline method declaration technique. Here you're using `() =>` and on line 18 above you're using `function()`

Reply:
Going to settle on `() =>`.